sql/create_train_val_test.go

```diff
@@ -39,20 +39,28 @@ func TestCreateTrainAndValDataset(t *testing.T) {
 
 	switch testDB.driverName {
 	case "maxcompute":
+		_, e := newTrainAndValDataset(testDB, testMaxcomputeTrainAndValDataset, "orig", 1)
+		a.Error(e)
+		_, e = newTrainAndValDataset(testDB, testMaxcomputeTrainAndValDataset, "orig", 0)
+		a.Error(e)
+		ds, e := newTrainAndValDataset(testDB, testMaxcomputeTrainAndValDataset, "orig", 0.8)
+		a.NoError(e)
+		a.Empty(ds.database)
+	case "hive":
 		_, e := newTrainAndValDataset(testDB, testTrainAndValDataset, "orig", 1)
 		a.Error(e)
 		_, e = newTrainAndValDataset(testDB, testTrainAndValDataset, "orig", 0)
 		a.Error(e)
 		ds, e := newTrainAndValDataset(testDB, testTrainAndValDataset, "orig", 0.8)
 		a.NoError(e)
-		a.True(ds.supported)
-	case "hive":
-		_, e := newTrainAndValDataset(testDB, testHiveTrainAndValDataset, "orig", 1)
+		a.NotEmpty(ds.database)
+	case "mysql":
```

Comment:
Is the case `"mysql"` identical to the case `"hive"`, if so, how about using the shorthand `case "hive", "mysql":`?